src/events/utils/event-translator.ts

```diff
@@ -7,13 +7,17 @@ import { SerializedEvent } from '../interfaces/serialized-event';
 export function serializedEventFromRecordWithMetadata(
   event: EventWithMetadata,
 ): SerializedEvent {
+  const metadata = event.metadata;
```

Comment:
Can you add the URL [here](https://github.com/iron-fish/ironfish-api/blob/master/src/events/events.service.ts#L69-L76)?